vrx_gz/src/Wavefield.cc

```diff
@@ -469,22 +471,48 @@ void Wavefield::Load(const msgs::Param &_msg)
 {
   auto params = _msg.params();
 
-  this->data->size = {params["size"].vector3d_value().x(),
-    params["size"].vector3d_value().y()};
-  this->data->cellCount = {params["cell_count"].vector3d_value().x(),
-    params["cell_count"].vector3d_value().y()};
-  this->data->number = params["number"].int_value();
-  this->data->scale = params["scale"].double_value();
-  this->data->angle = params["angle"].double_value();
-  this->data->steepness = params["steepness"].double_value();
-  this->data->amplitude = params["amplitude"].double_value();
-  this->data->period = params["period"].double_value();
-  this->data->phase = params["phase"].double_value();
-  this->data->direction = {params["direction"].vector3d_value().x(),
-    params["direction"].vector3d_value().y()};
-  this->data->model = params["model"].string_value();
-  this->data->gain = params["gain"].double_value();
-  this->data->tau = params["tau"].double_value();
+  if (params.count("size") > 0){
```

Comment:
nitpick: Could you move the open bracket to the next line to be consistent with the Gazebo style?

```
if (params.count("size") > 0)
{
  this->data->size = {params["size"].vector3d_value().x(),
    params["size"].vector3d_value().y()};
}
```

Reply:
sure, i will update them